### PR TITLE
No need to declare 2 routes, 1 is enough.

### PR DESCRIPTION
**Pull request by lucasmichot**
- [ ] Added or updated tests
- [ ] Added Docs for all relevant versions
- [x] Updated the changelog

**Related Issue/Intent**
No need to declare 2 routes, 1 is enough.

---

**Changes**
None

---

**Breaking changes**
None

### CODE REVIEW

**Comment by spawnia** [comment]
Does this method have an advantage?

**Comment by lucasmichot** [comment]
Well it does, this way we only have one route declared instead of 2 duplicate routes, with 2 different verbs

File: src/Support/Http/routes.php [hunk 1]
```diff
@@ -4,14 +4,11 @@
     $routeName = config('lighthouse.route_name', 'graphql');
     $controller = config('lighthouse.controller');
 
-    if (config('lighthouse.route_enable_get', false)) {
-        app('router')->get($routeName, [
-            'as' => 'lighthouse.graphql',
-            'uses' => $controller,
-        ]);
-    }
+    $methods = config('lighthouse.route_enable_get', false) ?
```
Comment (spawnia) [inline src/Support/Http/routes.php]: We do ternarys with a leading operator.

Comment (lucasmichot) [reply src/Support/Http/routes.php]: Changed in https://github.com/nuwave/lighthouse/pull/738/commits/472f62a7529e7a93a5d0adae7b33e73ec81da8e1

**Comment by spawnia** [comment]
Can you update the changelog please? I think this alters the behaviour of Lighthouse slightly

**Comment by spawnia** [comment]
Thanks @lucasmichot

**Pull request merged**